timesketch/frontend-ng/src/components/Explore/EventList.vue

```diff
@@ -15,6 +15,17 @@ limitations under the License.
 -->
 <template>
   <div>
+    <v-dialog v-model="fullscreenLoading" width="700">
+      <v-card flat class="pa-5">
+        <v-toolbar flat>
+          <v-progress-circular indeterminate size="20" width="1"></v-progress-circular>
+          <span class="ml-5">Exporting {{ totalHits }} events</span>
+          <v-spacer></v-spacer>
+          <v-btn depressed @click="fullscreenLoading = false">cancel</v-btn>
```

Comment:
Does "canceling" the export dialog stop the export process in any way?
I noticed that my timesketch instance gets very slow and/or freezes when I try (or accidentally) export >150k events. Even when I hit cancel right away.

Reply:
I removed the cancel function for now.

Reply:
Yeah, removing the cancel function does make sense to me as long as there will be side-effects when canceling.